tutorials/iot/ubuntu-kiosk/x11-kiosk.md

```diff
@@ -234,26 +234,32 @@ base: core18
 confinement: strict
 grade: devel
 
-apps:
-  glxgears-example:
-    command: xwayland-kiosk-launch glxgears
-    environment:
-      XWAYLAND_FULLSCREEN_WINDOW_HINT: title="glxgears"
-    plugs:
-      - opengl
-      - wayland
+layout:
+  /usr/share/X11:
+    bind: $SNAP/usr/share/X11
```

Comment:
Could you briefly explain in the tutorial what is this for? And link to https://snapcraft.io/docs/snap-layouts?